### PR TITLE
Added Drakeborgs for sillicons along with plushies

### PR DESCRIPTION
**Pull request by SarmentiCampbell**
Adds drakeborgs sprites for dogborgs
b57f283e5ceea2e0b6d9b0a637a38faa
Along with two new plushies.

Credit to Navier#1236 for the drakeborgs.
[Link to the License](https://github.com/Strazyplus/Skyrat13/blob/d68ef19207d1d79400d3bf671d862d18f0f0a822/icons/mob/cyborg/Drakeborg-licensing.txt)

### CODE REVIEW

**Comment by Iroquois-Pliskin** [comment]
No thanks.

**Comment by Heroman3003** [comment]
Been hoping for more variety for a while. This is good.

**Review by Novacat — COMMENTED** [review]
Only issue I can see offhand is that the plushies will not be obtainable, but that is fairly minor. Verk should probably review this.

**Review by Novacat — CHANGES_REQUESTED** [review]
Oh yeah, going to need proper licensing stuff.

**Comment by Verkister** [comment]
Looks pretty good to me, just the plushies seem unobtainable at the moment and the non-medical bellysprites no longer require the _g variants.

Also oof I just also noticed a blooper of my own that would be nice and simple to see fixed along this to avoid dmi conflicts. the sec otieborg's wreck sprite seems to be marked as a movement state, which it shouldn't be.

**Comment by SarmentiCampbell** [comment]
I assumed that peoples could get it from the plushie selection in the loadout, Will do that

**Review by Novacat — CHANGES_REQUESTED** [review]
I think you might have forgotten to commit the license itself? This seems otherwise fine.

File: code/modules/mob/living/silicon/robot/robot_vr.dm [hunk 1]
```diff
@@ -142,6 +142,10 @@
 
 /mob/living/silicon/robot/proc/vr_sprite_check()
 	if(wideborg == TRUE)
+		if(icontype== "Drake") // Why, Why can't we have normal nice things
+			icon = 'icons/mob/drakeborg/drakeborg_vr.dmi' 
+		else
+			icon = 'icons/mob/widerobot_vr.dmi'
```
Comment (SplinterGP) [inline code/modules/mob/living/silicon/robot/robot_vr.dm:148]: This else is not really needed, it worked without that

Comment (Verkister) [reply code/modules/mob/living/silicon/robot/robot_vr.dm:148]: the else is for the non-drakeborgs

**Pull request merged**